src/main.rs

```diff
@@ -65,9 +68,13 @@ pub fn run(prototype: Option<usize>) {
     let mut app = Application::new(graphics, &window);
     if let Some(brick_index) = prototype {
         app.capture_prototype(brick_index);
-    } else {
+    } else if let Some(_experiment_number) = experiment {
```

Comment:
What's your intention with this? Mapping an experiment number to a PostIteration op?

Reply:
It's kind of still a sketch.  I want to do experiments (like the inside-out rolling donut) which will need some hard-coding because it's too weird to capture in tenscript.

Reply:
If so, better to make a rich enum, let `clap` handle the enum variants and forget everything I wrote about `dyn PostIterate`. Just write one big `post_iterate` handler